src/templates/hi/2019/base.html

```diff
@@ -0,0 +1,188 @@
+{% extends "base/2019/base.html" %}
+
+{% block description %}The Web Almanac वेब रिपोर्ट की एक वार्षिक स्थिति है जो वेब समुदाय की विशेषज्ञता को HTTP संग्रह के डेटा और रुझानों के साथ जोड़ती है।{% endblock %}
```

Comment:
Should "The" be translated?

Reply:
"The" article doesn't make sense in Hindi Literature
Here we have two options we can ignore "The" as I did and also we can remove "The" from here.

Let's take an Example

The Taj Mahal (In English)
ताजमहल (In Hindi)

Reply:
Will take your guidance on this as to what reads best. Here's what we said in the [Translator's Guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Translators'-Guide):

> English product names (Web Almanac, HTTP Archive) should remain in English but can be adjusted slightly to local version (e.g. Le Web Almanac rather than The Web Almanac). For some languages this rule may be ignored if there are language specific character sets (e.g. Japanese might use katakana, hiragana, or kanj rather than English if that makes more sense).

Reply:
Yes, I have read that. According to my "Web Almanac" would be best here.
Because "The" article doesn't make sense in Hindi Literature.

Reply:
Sounds good to me!

Reply:
Great will do it :)